src/assets/scripts/client/airport/AirportModel.js

```diff
@@ -514,7 +514,6 @@ export default class AirportModel {
         .done((data) => {
             try {
                 // eslint-disable-next-line no-undef
-                log('Parsing terrain');
```

Comment:
as much as I'd love to see this go away, could you add this log statement back in?